tut-create-video.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: How do I create a video of my trace?
+sidebar: Create a Video?
+order: 70
+permalink: /tutorials/createvideo
+parent: tutorials
+---
+
+GAPID has the ability to export a captured trace as a video.
+
+This feature is only available via the command line tool `gapit`.
+
+### Prerequisites
+* A previously captured trace file ending in *.gfxtrace
+* Either avconv or ffmpeg encoding libraries on your path (for regular video encoding)
+
+<div class="callouts" markdown="block">
+
+1. Launch the command line interface.
+
+2. Note the path in which gapid was installed.
+<span class="info">e.g. `C:\Program Files (x86)\gapid\` on Windows; `/Applications/GAPID.app/Contents/MacOS/` on Mac OS X; or `/opt/gapid/` on linux.</span>
+
+3. Run the following command
+```
+$ <gapid_install_path>gapit video -type regular -out <output_path>.mp4 <path_to_tracefile>
```

Comment:
No need for `-type regular`, this is implicit, right?

Reply:
`-type auto` is the default, if the user somehow created their trace with FrameBufferObservations turned on then it would actually output a side by side video instead.

Reply:
Sure, but nobody would do that, right?

Reply:
(I'm only nitting on this because fewer options are less intimidating)

Reply:
Just wanted to make sure there were no surprises. I've removed it though.